packages/desktop-client/src/components/select/RecurringSchedulePicker.tsx

```diff
@@ -499,32 +515,33 @@ function RecurringScheduleTooltip({
               });
             }}
           />
-          <label
-            htmlFor="form_skipwe"
-            style={{
-              userSelect: 'none',
-              marginRight: 5,
-            }}
-          >
-            {t('Move schedule')}{' '}
-          </label>
-          <Select
-            id="solve_dropdown"
-            options={[
-              ['before', 'before'],
-              ['after', 'after'],
-            ]}
-            value={state.config.weekendSolveMode}
-            onChange={value => dispatch({ type: 'set-weekend-solve', value })}
-            disabled={!skipWeekend}
-          />
-          <label
-            htmlFor="solve_dropdown"
-            style={{ userSelect: 'none', marginLeft: 5 }}
-          >
-            {' '}
-            {t('weekend')}
-          </label>
+          <Trans>
+            <label
+              htmlFor="form_skipwe"
+              style={{
+                userSelect: 'none',
+                marginRight: 5,
+              }}
+            >
+              Move schedule{' '}
+            </label>
+            <Select
+              id="solve_dropdown"
+              options={[
+                ['before', 'before'],
+                ['after', 'after'],
+              ]}
+              value={state.config.weekendSolveMode}
+              onChange={value => dispatch({ type: 'set-weekend-solve', value })}
+              disabled={!skipWeekend}
+            />
+            <label
+              htmlFor="solve_dropdown"
+              style={{ userSelect: 'none', marginLeft: 5 }}
+            >
+              <Trans> {{ beforeOrAfter: '' }} weekend</Trans>
```

Comment:
Clever! I wonder if there's some way we can generalise this, I think there a lot of places in the app where we have an English sentence with dynamic portions

Reply:
Maybe we could have some sort of inverted pattern for a `Select` where we do
```tsx
<TransSelect options={[
  <Trans>Move schedule <Item>before</Item> weekend</Trans>,
  <Trans>Move schedule <Item>after</Item> weekend,
]}/>
```

The rationale with them being separate strings is this expands better to more options, and I'm not sure if there are languages where the position of the word could vary depending on the word.

But just food for thought! We don't have to implement this now